simple.go

```diff
@@ -22,57 +23,41 @@ type proofVerifier struct{}
-	var commRa, commDa, ticketa, seeda [32]byte
-	copy(commRa[:], commR)
-	copy(commDa[:], commD)
-	copy(ticketa[:], ticket)
-	copy(seeda[:], seed)
-	proverIDa := addressToProverID(proverID)
-
-	return ffi.VerifySeal(sectorSize, commRa, commDa, proverIDa, ticketa, seeda, sectorNum, proof)
+func (proofVerifier) VerifySeal(proofType abi.RegisteredProof, sealedCID, unsealedCID cid.Cid, prover address.Address, ticket abi.SealRandomness, seed abi.InteractiveSealRandomness, sectorNum abi.SectorNumber, proof abi.SealProof) (bool, error) {
+	return ffi.VerifySeal(proofType, sealedCID, unsealedCID, addressToProverID(prover), ticket, seed, sectorNum, proof)
 }
 
-func (proofVerifier) VerifyElectionPost(ctx context.Context, sectorSize abi.SectorSize, sectorInfo SortedPublicSectorInfo, challengeSeed []byte, proof []byte, candidates []EPostCandidate, proverID address.Address) (bool, error) {
+func (proofVerifier) VerifyElectionPost(ctx context.Context, sectorInfo ffi.SortedPublicSectorInfo, challengeSeed []byte, proof []byte, candidates []abi.PoStCandidate, prover address.Address) (bool, error) {
 	challengeCount := ElectionPostChallengeCount(uint64(len(sectorInfo.Values())), 0)
-	return verifyPost(ctx, sectorSize, sectorInfo, challengeCount, challengeSeed, proof, candidates, proverID)
+	return verifyPost(ctx, sectorInfo, challengeCount, challengeSeed, proof, candidates, prover)
 }
 
-func (proofVerifier) VerifyFallbackPost(ctx context.Context, sectorSize abi.SectorSize, sectorInfo SortedPublicSectorInfo, challengeSeed []byte, proof []byte, candidates []EPostCandidate, proverID address.Address, faults uint64) (bool, error) {
+func (proofVerifier) VerifyFallbackPost(ctx context.Context, sectorInfo ffi.SortedPublicSectorInfo, challengeSeed []byte, proof []byte, candidates []abi.PoStCandidate, prover address.Address, faults uint64) (bool, error) {
 	challengeCount := fallbackPostChallengeCount(uint64(len(sectorInfo.Values())), faults)
-	return verifyPost(ctx, sectorSize, sectorInfo, challengeCount, challengeSeed, proof, candidates, proverID)
+	return verifyPost(ctx, sectorInfo, challengeCount, challengeSeed, proof, candidates, prover)
 }
 
-func verifyPost(ctx context.Context, sectorSize abi.SectorSize, sectorInfo SortedPublicSectorInfo, challengeCount uint64, challengeSeed []byte, proof []byte, candidates []EPostCandidate, proverID address.Address) (bool, error) {
-	var challengeSeeda [CommLen]byte
-	copy(challengeSeeda[:], challengeSeed)
-
+func verifyPost(ctx context.Context, sectorInfo ffi.SortedPublicSectorInfo, challengeCount uint64, challengeSeed abi.PoStRandomness, proof []byte, candidates []abi.PoStCandidate, prover address.Address) (bool, error) {
 	_, span := trace.StartSpan(ctx, "VerifyPoSt")
 	defer span.End()
-	prover := addressToProverID(proverID)
-	return ffi.VerifyPoSt(sectorSize, sectorInfo, challengeSeeda, challengeCount, proof, candidates, prover)
-}
 
-func NewSortedPublicSectorInfo(sectors []ffi.PublicSectorInfo) SortedPublicSectorInfo {
```

Comment:
Is the idea that consumers will call this method directly on ffi? I guess that makes sense. Honestly, I'm not crazy about losing the abstraction between sectorbuilder consumers and ffi, even if they need to depend on ffi anyway.